mix.exs

```diff
@@ -36,7 +36,6 @@ defmodule OMG.Umbrella.MixProject do
       # NOTE: we're overriding for the sake of `omg_api` mix.exs deps. Otherwise the override is ignored
       # TODO: removing the override is advised, but it gives undefined symbol errors, see
       #       https://github.com/exthereum/exth_crypto/issues/8#issuecomment-416227176
-      {:libsecp256k1, "~> 0.1.4", compile: "${HOME}/.mix/rebar compile", override: true},
```

Comment:
there's a comment relevant to that line just above. If the override is unnecessary, and you're removing it, the dangling comment is horrible, please be careful